cmd/release/cmd/inspect.go

```diff
@@ -0,0 +1,136 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log/slog"
+	"os"
+	"text/tabwriter"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	reg "github.com/rancher/ecm-distro-tools/registry"
+	"github.com/rancher/ecm-distro-tools/release"
+	"github.com/rancher/ecm-distro-tools/release/rke2"
+	"github.com/rancher/ecm-distro-tools/repository"
+	"github.com/spf13/cobra"
+)
+
+const (
+	ossRegistry = "docker.io"
+)
+
+func displayResults(results []rke2.ImageStatus) error {
+	w := tabwriter.NewWriter(os.Stdout, 0, 8, 2, ' ', 0)
+	defer w.Flush()
+
+	fmt.Fprintln(w, "image\toss\tprime\tsig\tamd64\tarm64\twin")
+	fmt.Fprintln(w, "-----\t---\t-----\t---\t-----\t-----\t-------")
+
+	for _, result := range results {
+		ossStatus := boolToMark(result.OSSImage.Exists)
+		primeStatus := "✗"
+		if result.PrimeImage.Exists {
+			primeStatus = "✓"
+		}
+
+		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\t%s\t%s\n",
+			formatImageRef(result.Reference),
+			ossStatus,
+			primeStatus,
+			"?", // sigstore not implemented
+			archStatus(result.ExpectsLinuxAmd64, result.OSSImage, result.PrimeImage, reg.Platform{OS: "linux", Architecture: "amd64"}),
+			archStatus(result.ExpectsLinuxArm64, result.OSSImage, result.PrimeImage, reg.Platform{OS: "linux", Architecture: "arm64"}),
+			windowsStatus(result.ExpectsWindows, result.OSSImage.Exists && result.PrimeImage.Exists),
+		)
+	}
+
+	return nil
+}
+
+func archStatus(expected bool, ossInfo, primeInfo reg.Image, platform reg.Platform) string {
+	if !expected {
+		return "-"
+	}
+
+	hasArch := ossInfo.Platforms[platform] && primeInfo.Platforms[platform]
+	if hasArch {
+		return "✓"
+	}
+	return "✗"
+}
+
+func windowsStatus(expected, exists bool) string {
+	if !expected {
+		return "-"
+	}
+	if exists {
+		return "✓"
+	}
+	return "✗"
+}
+
+func formatImageRef(ref name.Reference) string {
+	return fmt.Sprintf("%s:%s", ref.Context().RepositoryStr(), ref.Identifier())
+}
+
+func boolToMark(b bool) string {
+	if b {
+		return "✓"
+	}
+	return "✗"
+}
+
+var inspectCmd = &cobra.Command{
+	Use:   "inspect [version]",
+	Short: "Inspect release artifacts",
+	Long: `Inspect release artifacts for a given version.
+Currently supports inspecting the image list for published rke2 releases.`,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		// Set up slog configuration based on debug flag
+		var logLevel slog.Level
```

Comment:
Are we using slog in the other commands? If not, we should do whatever the others are doing for consistency.

Reply:
`debug` is an unused option, there's no existing implementation of debug logging at all. removed all logging.